assets/xml/objects/object_zl1.xml

```diff
@@ -1,90 +1,114 @@
 ﻿<Root>
     <File Name="object_zl1" Segment="6">
```

Comment:
```suggestion
﻿<Root>
﻿    <!-- Object for Child Zelda 1 -->
    <File Name="object_zl1" Segment="6">
```

Reply:
Except... this is the unused object, It gets mentioned in the empty actor ovl_En_Zl1, but that actor doesn't use the object at all with an empty draw function and no references to the contents of this object.

Should we include "Unused" in the description comment?